notebooks/04_model_select_and_optimize/hypertune_spark_deep_dive.ipynb

```diff
@@ -650,7 +1152,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 78,
+   "execution_count": 65,
    "metadata": {},
    "outputs": [],
    "source": [
```

Comment:
The first time I read the notebook, the title `5 Hyperparameter tuning with `hyperopt` sampling methods` was weird to me, since the previous one was `4 Hyperparameter tuning with hyperopt`. Then I came back to 4 and realized that there we were using TPE. 
Maybe we could change 4 to be something like `4 Hyperparameter tuning with hyperopt TPE methods`?. Another option could be to have a big section of hyperopt, and then 2 subsections with the two methods.